app/pages/lab/help/how-to-page.cjsx

```diff
@@ -244,6 +244,46 @@ counterpart.registerTranslations 'en',
       You can create a manifest file in a simple text editor (such as TextEdit or Notepad), although this method is prone to errors like missed or extra commas. People often find it easiest to create manifest files using spreadsheet software such as Google Sheets, iWork Numbers or Microsoft Excel. Creating and maintaining a manifest using a spreadsheet makes the manifest easy to read, and you can export it to CSV format when you're ready to upload your subjects. You can also open existing CSV files in spreadsheet software.
 
       *Note:* if you have a large subject set it may be cumbersome to manually create a manifest. We suggest using a command-line or other tool to copy-paste a directory list of files into a spreadsheet to help you get started.
+
+    ### Generalised Feedback
+
+      Feedback is enabled per-task - there's a checkbox under the answer editing box that toggles it on and off.
+
+      ![screen shot 2017-04-13 at 14 29 36](https://cloud.githubusercontent.com/assets/2725841/25006659/b8d81398-2055-11e7-8cb6-8c660b4173b7.png)
```

Comment:
The alt text here ('screen shot 2017-04-13 at 14 29 36') should describe the content of the image. Maybe a short description of what the screenshot shows?

Reply:
Should the images be added to PFE, instead of pulled in from github?